st2common/MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+# https://docs.python.org/2/distutils/sourcedist.html#commands
+# Include all files under the source tree by default.
+# Other behaviour can be used though.
+recursive-include st2common *.* *
```

Comment:
I'm still confused by that line - "st2common/{*,*.*}" basically means everything under st2common/st2common.

In any case, like I said above, I would prefer to be explicit and collect Python modules and packages we want to include inside setup.py and only explicitly declare other non-package files we want to include here.